ts/features/newWallet/screens/WalletHomeScreen.tsx

```diff
@@ -49,16 +51,62 @@ const WalletHomeScreen = ({ route }: Props) => {
     }, [isNewElementAdded])
   );
 
+  /* CODE RELATED TO THE HEADER */
+
+  const navigation = useIONavigation();
+  const scrollViewContentRef = useAnimatedRef<Animated.ScrollView>();
+
+  /* Scroll to top when the active tab is tapped */
+  useTabItemPressWhenScreenActive(
+    () => scrollViewContentRef.current?.scrollTo({ y: 0, animated: true }),
+    false
+  );
+
+  const isSettingsVisibleAndHideProfile = useIOSelector(
+    isSettingsVisibleAndHideProfileSelector
+  );
+
+  const helpAction = useHeaderFirstLevelActionPropHelp(ROUTES.WALLET_HOME);
+  const settingsAction = useHeaderFirstLevelActionPropSettings();
+
+  useLayoutEffect(() => {
+    const headerFirstLevelProps: HeaderFirstLevel = {
+      title: I18n.t("wallet.wallet"),
+      firstAction: helpAction,
+      testID: "wallet-home-header-title",
+      animatedRef: scrollViewContentRef,
+      ...(isSettingsVisibleAndHideProfile
+        ? {
+            type: "twoActions",
+            secondAction: settingsAction
+          }
+        : { type: "singleAction" })
+    };
+
+    navigation.setOptions({
+      header: () => <HeaderFirstLevel {...headerFirstLevelProps} />
+    });
+  }, [
+    helpAction,
+    isSettingsVisibleAndHideProfile,
+    navigation,
+    scrollViewContentRef,
+    settingsAction
+  ]);
+
```

Comment:
We could move everything inside the `WalletScrollView` component. What do you think?

Reply:
Done in the following commit → https://github.com/pagopa/io-app/pull/6054/commits/4005f91fb2ae8e4c8a58a5c9bd15e60e0ddbb6ae